libs/partners/databricks/pyproject.toml

```diff
@@ -0,0 +1,99 @@
+[tool.poetry]
+name = "langchain-databricks"
+version = "0.1.0"
+description = "An integration package connecting Databricks and LangChain"
+authors = []
+readme = "README.md"
+repository = "https://github.com/langchain-ai/langchain"
+license = "MIT"
+
+[tool.poetry.urls]
+"Source Code" = "https://github.com/langchain-ai/langchain/tree/master/libs/partners/databricks"
+"Release Notes" = "https://github.com/langchain-ai/langchain/releases?q=tag%3A%22databricks%3D%3D0%22&expanded=true"
+
+[tool.poetry.dependencies]
+# TODO: Replace <3.12 to <4.0 once https://github.com/mlflow/mlflow/commit/04370119fcc1b2ccdbcd9a50198ab00566d58cd2 is released
```

Comment:
note: MLflow currently uses `pkg_resources` (removed in the PR in the comment) so does not support Python 3.12. Once we release a new version contains the PR, we can remove this pin.